tremor-cli/tests/integration/system_metrics/metrics.trickle

```diff
@@ -4,7 +4,7 @@ script
 
   let state = match state of
     # exit when we have all metrics events
-    case %{ present onramp, present pipeline, present offramp } when state.onramp >= 9 and state.pipeline >= 4 and state.offramp >= 6 => emit {"exit": 0, "delay": 100} => "done"
+    case %{ present onramp, present `pipeline`, present offramp } when state.onramp >= 9 and state.`pipeline` >= 4 and state.offramp >= 6 => emit {"exit": 0, "delay": 100} => "done"
```

Comment:
TODO: Not a comment but a mental note to add `pipeline` as a non backward compatible change for the changelog.

Reply:
better captured elsewhere